Clients/src/presentation/components/CreateProjectForm/styles.ts

```diff
@@ -0,0 +1,9 @@
+export const styles = {
+    textarea: {
+      maxHeight: 101,
+      height: 101
+    },
+    select: {
+      mt: 11.5
```

Comment:
instead of values like `11.5`, please use theme value, `theme.spacing( integer value )`